src/pages/data-management/index.jsx

```diff
@@ -112,7 +104,7 @@ const DataManagementPage = () => {
           );
         }
 
-        return (<ProjectDetails width={width} height={height} />);
```

Comment:
why are these removed?

Reply:
The `width` and `height` properties were used in this component to calculate the height of project details component. I replaced this with css changes to the `ProjectDetails.jsx` and `MultiTIleContainer.jsx` to provide the same effect.